[1: 1516567238.659557s]
init_syscontext() C-kern/platform/Linux/init.c:103
Exit function with
Error 22 - Invalid argument
[1: 1516567238.659561s]
init_syscontext() C-kern/platform/Linux/init.c:103
Exit function with
Error 22 - Invalid argument
[1: 1516567238.659562s]
init_syscontext() C-kern/platform/Linux/init.c:103
Exit function with
Error 22 - Invalid argument
[1: 1516567238.659767s]
init_syscontext() C-kern/platform/Linux/init.c:103
Exit function with
Error 22 - Invalid argument
[1: 1516567238.659769s]
initrun_syscontext() C-kern/platform/Linux/init.c:197
Exit function with
Error 22 - Invalid argument
[1: 1516567238.659771s]
init_syscontext() C-kern/platform/Linux/init.c:103
Exit function with
Error 22 - Invalid argument
[1: 1516567238.659772s]
initrun_syscontext() C-kern/platform/Linux/init.c:197
Exit function with
Error 22 - Invalid argument
[1: 1516567238.659774s]
init_syscontext() C-kern/platform/Linux/init.c:103
Exit function with
Error 22 - Invalid argument
[1: 1516567238.659775s]
initrun_syscontext() C-kern/platform/Linux/init.c:197
Exit function with
Error 22 - Invalid argument
[1: 1516567238.659782s]
initrun_syscontext() C-kern/platform/Linux/init.c:122
Function call 'new_threadlocalstore' failed with error 333
Exit function with
Error 333 - Unknown error
[1: 1516567238.659791s]
initrun_syscontext() C-kern/platform/Linux/init.c:133
System call 'sigaltstack' failed with error 333
Exit function with
Error 333 - Unknown error
[1: 1516567238.659801s]
initrun_syscontext() C-kern/platform/Linux/init.c:143
System call 'getcontext' failed with error 333
Exit function with
Error 333 - Unknown error
[1: 1516567238.659811s]
initrun_syscontext() C-kern/platform/Linux/init.c:161
System call 'swapcontext' failed with error 333
Exit function with
Error 333 - Unknown error
[1: 1516567238.659822s]
initrun_syscontext() C-kern/platform/Linux/init.c:174
System call 'sigaltstack' failed with error 333
Exit function with
Error 333 - Unknown error
[1: 1516567238.659835s]
initrun_syscontext() C-kern/platform/Linux/init.c:181
Function call 'delete_threadlocalstore' failed with error 333
Exit function with
Error 333 - Unknown error
[0: 1516567238.660867s]
callmain_platform() C-kern/platform/Linux/init.c:64
Abort main thread '0' with fatal error
Exit function with
Error 131 - State not recoverable
[0: 1516567238.660885s]
abort_maincontext() C-kern/main/maincontext.c:275
Abort process with fatal error
Error 131 - State not recoverable
